src/vmm/src/resources.rs

```diff
@@ -275,6 +288,14 @@ impl VmResources {
         self.vsock.insert(config)
     }
 
+    /// Sets a balloon device to be attached when the VM starts.
+    pub fn set_balloon_device(
```

Comment:
Do we actually need this fn? Why not just call ballon.insert directly ?

Reply:
It currently mirrors the interface of all other devices. It is not *needed* but it looks symmetrical.

Reply:
I think it is more important to make it simpler than symmetrical.

Reply:
I made it simpler rather than symmetrical.